web/src/message_events.js

```diff
@@ -77,6 +81,7 @@ function maybe_add_narrowed_messages(messages, msg_list, callback, attempt = 1)
 
             callback(new_messages, msg_list);
             unread_ops.process_visible();
+            compose_notifications.notify_messages_outside_current_search(elsewhere_messages);
```

Comment:
I notice this doesn't include the `narrow.ts` change and linked `message_events` call from https://github.com/zulip/zulip/commit/a2ef1d7e9376de270be7537472f937b335119e53 in `message_view.js`. Is that an intentional decision?